test/helpers/visit.js

```diff
@@ -1,49 +0,0 @@
-import nightmare from 'nightmare';
-import url from 'url';
```

Comment:
FYI, I never found the `url` package, so I assume that it has been removed already.